### PR TITLE
add `en_kawaii` locale

### PR DESCRIPTION
**Pull request by FjorgeTheHuman**
Disclaimer: Cuteness levels not measured

### CODE REVIEW

File: lib/locales/en_kawaii.yml [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+        success: "Bye (╥︣﹏╥᷅). I'll start talking to you in `%s`."
+
+  cfg:
+    usage: "Usage:"
+    subcmd: "subcommand"
+    options: "options"
+    avail_subcmd: "Available subcommands:"
+
+    help:
+      help: "You already know what does does, dummy! (๑˃́ꇴ˂̀)"
```
Comment (anna328p) [inline lib/locales/en_kawaii.yml:187]: I don't understand

File: lib/locales/en_kawaii.yml [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+  xkcd:
+
+  util:
+    invite:
+      title: "Invite %s to your server! (^_^)"
+      desc: "[Click this link to invite %s to your server](%s) so we can be together even more (^◡^)"
+
+  uc:
+    help:
+      help: "You already know what does does, dummy! (๑˃́ꇴ˂̀)"
```
Comment (anna328p) [inline lib/locales/en_kawaii.yml:161]: Same thing here

File: lib/locales/en_kawaii.yml [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+      last-update: "last updated"
+      link: "link"
+
+  blacklist:
+    bl:
+      title: Blacklist entries
+    message: >-
+      Sorry I can"t let you send that (>.<) `%s`
+
+      Here"s what you sent though, so pwease don"t do it next time! (^_^)
```
Comment (anna328p) [inline lib/locales/en_kawaii.yml:81]: wrong quote

**Review by anna328p — CHANGES_REQUESTED** [review]

**Review by anna328p — APPROVED** [review]

**Comment by FjorgeTheHuman** [comment]
I've fixed the commit names now

**Pull request merged**